src/recommendation/src/test/scala/SARSpec.scala

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import scala.language.existentials
+
+class SARSpec extends RankingTestBase with EstimatorFuzzing[SAR] {
+  override def testObjects(): List[TestObject[SAR]] = {
```

Comment:
Can you add some tests that show that it improves over ALS on a dataset. This way we could sanity check. Also if there's a benchmark in a paper we could replicate that would be reassuring